app/src/layouts/DashboardLayout/Sidebar/SideBarHeader/index.tsx

```diff
@@ -23,7 +26,7 @@ const SideBarHeader: React.FC<SideBarHeaderProps> = ({
     trackSidebarClicked("collapse_button");
   };
 
-  const trigger = createElement(collapsed ? RiMenuUnfoldLine : RiMenuFoldLine, {
+  const trigger = createElement(collapsed ? RightChevron : LeftChevron, {
```

Comment:
Since `RightChevron` & `LeftChevron` are react components so no need to pass it again to `createElement`.
```suggestion
 <Button
    type="text"
    icon={collapsed ? 
           <RightChevron className="trigger"/> :  
           <LeftChevron className="trigger"/>}
    onClick={toggle}
    className="siderbar-collapse-btn"
    />
```